### DISCUSSION TITLE
Regarding Error e3sm.bldlog file.

### DISCUSSION THREAD

**Discussion by psalunke9** [E3SM-Project/e3sm_diags#888] [Q&A]
Hello,

I am attempting to set up the E3SM model.  I successfully finished certain building phases such as (gptl.bldlog, mct.bldlog, pio.bldlog, csm_share_bldlog, kokkos.bldlog, etc.). I encountered the following error in (e3sm.bldlog).

Could someone please assist me?

Please see error below:

2024-11-04 10:55:50 atm 
Create namelist for component eam
   Calling /net/fs12/d0/psalunke/E3SM/components/eam//cime_config/buildnml
  2024-11-04 10:55:50 lnd 
Create namelist for component elm
   Calling /net/fs12/d0/psalunke/E3SM/components/elm//cime_config/buildnml
  2024-11-04 10:55:50 ice 
Create namelist for component mpassi
   Calling /net/fs12/d0/psalunke/E3SM/components/mpas-seaice/cime_config/buildnml
  2024-11-04 10:55:51 ocn 
Create namelist for component docn
   Calling /net/fs12/d0/psalunke/E3SM/components/data_comps/docn/cime_config/buildnml
  2024-11-04 10:55:51 rof 
Create namelist for component mosart
   Calling /net/fs12/d0/psalunke/E3SM/components/mosart//cime_config/buildnml
  2024-11-04 10:55:51 glc 
Create namelist for component sglc
   Calling /net/fs12/d0/psalunke/E3SM/components/stub_comps/sglc/cime_config/buildnml
  2024-11-04 10:55:51 wav 
Create namelist for component swav
   Calling /net/fs12/d0/psalunke/E3SM/components/stub_comps/swav/cime_config/buildnml
  2024-11-04 10:55:51 iac 
Create namelist for component siac
   Calling /net/fs12/d0/psalunke/E3SM/components/stub_comps/siac/cime_config/buildnml
  2024-11-04 10:55:51 esp 
Create namelist for component sesp
   Calling /net/fs12/d0/psalunke/E3SM/components/stub_comps/sesp/cime_config/buildnml
  2024-11-04 10:55:51 cpl 
Create namelist for component drv
   Calling /net/fs12/d0/psalunke/E3SM/driver-mct/cime_config/buildnml
   NOTE: ignoring setting of rof2ocn_ice_rmapname=idmap in seq_maps.rc
   NOTE: ignoring setting of rof2ocn_liq_rmapname=idmap in seq_maps.rc
Building gptl with output to file /net/fs12/d0/psalunke/E3SM/cases/E3SM-TEST1/bld/gptl.bldlog.241104-105550
   Calling /net/fs12/d0/psalunke/E3SM/share/build/buildlib.gptl
Component gptl build complete with 1 warnings
Building mct with output to file /net/fs12/d0/psalunke/E3SM/cases/E3SM-TEST1/bld/mct.bldlog.241104-105550
   Calling /net/fs12/d0/psalunke/E3SM/cime/CIME/build_scripts/buildlib.mct
Building pio with output to file /net/fs12/d0/psalunke/E3SM/cases/E3SM-TEST1/bld/pio.bldlog.241104-105550
   Calling /net/fs12/d0/psalunke/E3SM/cime/CIME/build_scripts/buildlib.pio
Component pio build complete with 1 warnings
Building csm_share with output to file /net/fs12/d0/psalunke/E3SM/cases/E3SM-TEST1/bld/csm_share.bldlog.241104-105550
   Calling /net/fs12/d0/psalunke/E3SM/share/build/buildlib.csm_share
Component csm_share build complete with 20 warnings
Building kokkos with output to file /net/fs12/d0/psalunke/E3SM/cases/E3SM-TEST1/bld/kokkos.bldlog.241104-105550
   Calling /net/fs12/d0/psalunke/E3SM/cime/CIME/build_scripts/buildlib.kokkos
Configuring full e3sm model with output to file /net/fs12/d0/psalunke/E3SM/cases/E3SM-TEST1/bld/e3sm.bldlog.241104-105550
   Calling cmake directly, see top of log file for specific call
Building e3sm model with output to file /net/fs12/d0/psalunke/E3SM/cases/E3SM-TEST1/bld/e3sm.bldlog.241104-105550
   Calling make, see top of log file for specific call
ERROR: BUILD FAIL: build e3sm failed, cat /net/fs12/d0/psalunke/E3SM/cases/E3SM-TEST1/bld/e3sm.bldlog.241104-105550

Error you can see below:
vi /net/fs12/d0/psalunke/E3SM/cases/E3SM-TEST1/bld/e3sm.bldlog.241104-105550

**Target CMakeFiles/e3sm.exe.dir/net/fs12/d0/psalunke/E3SM/driver-mct/main/seq_map_type_mod.F90.o built in 0.531034 seconds
Target CMakeFiles/e3sm.exe.dir/net/fs12/d0/psalunke/E3SM/driver-mct/main/mrg_mod.F90.o built in 1.496372 seconds
gmake[2]: Leaving directory '/net/fs12/d0/psalunke/E3SM/cases/E3SM-TEST1/bld/cmake-bld'
CMakeFiles/Makefile2:660: recipe for target 'cmake/cpl/CMakeFiles/e3sm.exe.dir/all' failed
gmake[1]: *** [cmake/cpl/CMakeFiles/e3sm.exe.dir/all] Error 2
gmake[1]: Leaving directory '/net/fs12/d0/psalunke/E3SM/cases/E3SM-TEST1/bld/cmake-bld'
Makefile:93: recipe for target 'all' failed
gmake: *** [all] Error 2
real 476.36
user 1910.97
sys 243.20**

Thanks and Regards
Popat Salunke

**Comment by xylar** [comment]
@psalunke9, I believe you don't have the correct repository and your issue should be directed to:
https://github.com/E3SM-Project/E3SM/discussions

**Reply by psalunke9** [reply]
Hi,

Thank you for your quick response and sharing this link. 

Thanks and Regards,
Popat

**Comment by xylar** [comment]
Moved to https://github.com/E3SM-Project/E3SM/discussions/6732
